### DISCUSSION TITLE
education verify error

### DISCUSSION THREAD

**Discussion by Adeshen** [community/community#52299] [GitHub Education]
I have upload all the need education material change my profile to my real name
But it always says that 
"Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Apr 8, 2023 for [@Adeshen](https://github.com/Adeshen).

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio."

**Comment by googidaddy** [comment]
me too

**Reply by june-in-exile** [reply]
Same problem. But my classmate used similar document and passed. The verification mechanism is quite buggy.

**Comment by samus-aran** [comment]
Hi @Adeshen 👋 

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. This discussion was removed because it contained content that violates our Community Code of Conduct, including posting the [same comment in multiple places.](https://github.com/orgs/community/discussions/52300) As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
